_data/simple-icons.json

```diff
@@ -940,6 +940,11 @@
             "hex": "00A4DE",
             "source": "http://hatenacorp.jp/press/resource"
         },
+        {
+            "title": "Here",
```

Comment:
They seem to prefer the fully-capitalized "HERE" as their brand name

Reply:
Sure...